Lib/test/test_types.py

```diff
@@ -34,6 +38,28 @@ def clear_typing_caches():
 
 class TypesTests(unittest.TestCase):
 
+    def test_names(self):
```

Comment:
I don't remember how we usually call those test methods, but maybe `test_exported_names`?

Reply:
There's no pattern: `test_all`, `TestImportStar`, `test_module_attributes`, `test__all__`, `test_check_all`, `test_names`, etc.

Reply:
Oh I see. Nvm then, pick the one you want :')